src/integrationTest/java/uk/gov/hmcts/reform/demo/config/SwaggerPublisher.java

```diff
@@ -28,8 +27,6 @@
     private MockMvc mvc;
 
     @DisplayName("Generate swagger documentation")
-    @Test
```

Comment:
why remove `@Test` ?

Reply:
Because I am not sure it's a test based on the name of the class and the source code it doesn't look as a test.
It has not assertion, the name of the class is not having test in it
And anyway it's make fail integration

Reply:
Its intended to run as part of the test suite to generate swagger docs for publishing,

and it has that suppression as it doesn't have an assert but static analysis tools will complain